### DISCUSSION TITLE
All Graphs are incomprehensible when using multiple currencies

### DISCUSSION THREAD

**Discussion by cruzzoe** [firefly-iii/firefly-iii#8198] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)
- [X] My request is not listed as [a very good idea, but unfortunately...](https://docs.firefly-iii.org/firefly-iii/more-information/what-its-not/)
- [X] I've used [the search](https://github.com/firefly-iii/firefly-iii/issues?q=is%3Aissue) and this has not been requested before.

### Description

I have 2 currencies in my accounts: pounds (GBP) and yen (JPY).

When categorized, all my transactions (yen and GBP)  are placed onto the same graph. 

Todays exchange rate is roughly 183 yen to 1 gbp.

This means that comparing grocery shopping in Yen with a house purchase in Gbp appears to be equivilent! 

I understand FF3 will not do FX rate transactions and agree with the design decision.

### Solution

Is it possible we could have the category graph showing one currency at a time rather than all currencies on one graph?

This could be done by adding in a dropdown. Default graph should be that based on accounts default currency.

### What are alternatives?

None

**Comment by cruzzoe** [comment]
Note - the same issue affects lots of the other graphs too. E.g my revenue graphs don't make any sense because of the exchange rate problem (someone pays me value 10USD in jpy yen shows as 1800Yen whereas getting paid 100GBP (120USD) shows only as nominal value 100 despite the value being 10x greater. If my request is hard to implement, I'd really rather to make all my graphs support only ONE currency (where Iive) to avoid this confusion.

**Reply by mruizdia** [reply]
Hi, I've a similar issue, I just started using Firefly3, I came from KMymoney before.
Currently, I'm trying to record transactions in Euros, US Dollars and Argentinian Pesos, the problem I've is similar to yours, when I start recording the Argentinian pesos 1300 ARS to 1 EUR :( the graphs get crazy cause them shows the Argentinian Pesos as Euros.
Did you manage to find a solution?

I was trying to understand if it's possible to have a kind of conversion table that converts all the currencies into the default currency, but I think it's not such a possibility yet.
@JC5 Firefly 3 is amazing thanks for it, looking forward to having this issue solved and start supporting your project.

**Comment by cruzzoe** [comment]
Alternatively, I can run 1 instance of FF3 for Yen and one instance of FF3 for GBP currencies. It seems a shame to do this but right now, all the (lovely) graphs are a bit meaningless as the GBP values are completely dwarfed by Yen. Hence my UK balances hug the zero line in some of the graphs or don't feature in the picharts.

**Comment by JC5** [comment]
Yes, I know about this. It gets reported regularly, please use the search to find out more. I hope to fix this in a future release.
